views/index.html

```diff
@@ -95,31 +95,29 @@ <h1 class="animated fadeInUp">CVR</h1>
 <code>
 #!/bin/bash
 
-# Required Travis config:
-# - Add the repo token in the variable CVR_TOKEN as a secure property in the Travis configuration
```

Comment:
I suppose my inclination would be to replace "This script can be used to work with Travis, which doesn't make the token available on pull requests." with something like 

> This script can be used to work with Travis, passing the owner and repo instead of the token.

I'd also consider adding an additional `curl` example at the top that doesn't use the token, but that's beyond the scope of this PR.